integration-tests/pom.xml

```diff
@@ -142,6 +142,7 @@
         <module>smallrye-metrics</module>
         <module>smallrye-graphql</module>
         <module>smallrye-graphql-client</module>
+        <module>smallrye-opentracing</module>
```

Comment:
From looking at the integration test, I couldn't determine what's being tested as there's no verification of the trace being in Zipkin format or the like.

Reply:
I added an integration test that verifies the request reaches zipkin mock with the right format

Reply:
Apologies, I completely forgot to mention that for the integration test to run in native image, you need to update `.github/native-tests.json` to include the integration test module in one of the build job sections.

Reply:
np, I added to the file you mentioned. I hope I added in the right place :)

Reply:
I think that's fine